Procfile

```
@@ -0,0 +1 @@
+web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:development}
```

This has a typo and I think that's probably why it wouldn't push to heroku. It should be    You missed a minus sign before development 
```suggestion
web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development}
```